patches/chrome-app-app-Info.plist.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/app/app-Info.plist b/chrome/app/app-Info.plist
+index 9afdf482c5db005750c73bfa778629de2009921f..ee62b491774b8106150b5b1d08985f76c4c1f3c0 100644
+--- a/chrome/app/app-Info.plist
++++ b/chrome/app/app-Info.plist
+@@ -354,7 +354,7 @@
+ 		</dict>
+ 	</array>
+ 	<key>NSUserNotificationAlertStyle</key>
+-	<string>banner</string>
```

Comment:
see `brave_tweak_info_plist` for changing plist entries

Reply:
Fixed